src/utils/comic-issue.ts

```diff
@@ -14,8 +14,7 @@ export const getRarityShare = (numberOfCovers: number, rarity: ComicRarity) => {
   else {
     throw new Error('Unsupported number of rarities');
   }
-
-  return rarityShare.find((share) => share.rarity === rarity).value;
+  return rarityShare.find((share) => share.rarity.toString() === rarity).value;
```

Comment:
I just realized that I made a duplicate `getRarityShare` function in `constants.ts` and `utils/comic-issue.ts`...
Lets delete the one from constants and move the `getRarirtyShareTable` to utils as well